core/common/src/main/java/alluxio/exception/status/AlluxioStatusException.java

```diff
@@ -205,6 +205,25 @@ public static AlluxioStatusException fromCheckedException(Throwable throwable) {
   }
 
   /**
+   * Converts unchecked throwables to Alluxio status exceptions. Checked throwables should not be
+   * passed to this method.
+   *
+   * @param throwable a throwable
+   * @return the converted {@link AlluxioStatusException}
+   */
+  public static AlluxioStatusException fromUncheckedException(Throwable throwable) {
```

Comment:
does this need to be public? I would expect that public API to be `fromThrowable`